src/views/apply/Apply.tsx

```diff
@@ -28,22 +27,27 @@ function Apply() {
 		return null;
 	}
 
+	const applyBody = !acceptedPreface ? (
+		<ApplicationPreface
+			isLoggedIn={isLoggedIn}
+			setAcceptedPreface={setAcceptedPreface}
+		/>
+	) : (
+		<ApplicationForm setSubmittedApplication={setSubmittedApplication} />
+	);
+
 	return (
-		<div className={styles.main}>
+		<>
 			<Head>
 				<title>Apply | HackUCI 2023</title>
 			</Head>
-			<Container className={styles.apply}>
-				{!acceptedPreface ? (
-					<ApplicationPreface
-						isLoggedIn={isLoggedIn}
-						setAcceptedPreface={setAcceptedPreface}
-					/>
-				) : (
-					<ApplicationForm setSubmittedApplication={setSubmittedApplication} />
-				)}
+			<TitleBanner>
+				<h1>Apply</h1>
+			</TitleBanner>
```

Comment:
Might just be me, but the light is somewhat off here compared to the one on the home page:
![image](https://user-images.githubusercontent.com/28026520/212523980-b71783b2-132d-4457-a3d9-b03069bb7aa7.png)
vs.
![image](https://user-images.githubusercontent.com/28026520/212523990-b82e698f-4fc2-431e-81c2-b3caf68d113d.png)

Reply:
Maybe someone else could have a shot at this, but I'm not bothered enough to fix this right now 😣.